src/test/java/api/PrecedingSucceedingTitlesApiExamples.java

```diff
@@ -0,0 +1,496 @@
+      createdInstance.getId().toString(), null);
+    assertPrecedingSucceedingTitles(actualSucceedingTitle2, succeedingTitles.getJsonObject(1),
+      createdInstance.getId().toString(), null);
+  }
+
+  @Test
+  public void canUpdateAnInstanceWithUnconnectedPrecedingTitles()
+    throws InterruptedException, MalformedURLException, TimeoutException,
+    ExecutionException {
+
+    JsonObject precedingTitle = createSemanticWebUnconnectedTitle(UUID.randomUUID().toString());
+    JsonObject smallAngryPlanetJson = smallAngryPlanet(UUID.randomUUID());
+    smallAngryPlanetJson.put("precedingTitles", new JsonArray().add(precedingTitle));
+
+    IndividualResource createdInstance = instancesClient.create(smallAngryPlanetJson);
+
+    JsonObject newPrecedingTitle = new JsonObject()
+      .put("title", "New title")
+      .put("hrid", "inst000000000101")
+      .put("identifiers", new JsonArray().add(
+        new JsonObject()
+          .put("identifierTypeId", "1231054f-be78-422d-bd51-4ed9f33c3422")
+          .put("value", "000012104")));
+
+    JsonObject newInstance = createdInstance.getJson().copy();
+    newInstance.put("precedingTitles", new JsonArray().add(newPrecedingTitle));
+
+    instancesClient.replace(createdInstance.getId(), newInstance);
+
+    Response instanceResponse = instancesClient.getById(createdInstance.getId());
+    assertPrecedingSucceedingTitles(instanceResponse.getJson().getJsonArray("precedingTitles")
+        .getJsonObject(0), newPrecedingTitle, null,
+      createdInstance.getId().toString());
+  }
+
+  @Test
+  public void canUpdateAnInstanceWithUnconnectedSucceedingTitles()
+    throws InterruptedException, MalformedURLException, TimeoutException,
+    ExecutionException {
+
+    JsonObject succeedingTitle = createSemanticWebUnconnectedTitle(UUID.randomUUID().toString());
+    JsonObject smallAngryPlanetJson = smallAngryPlanet(UUID.randomUUID());
+    smallAngryPlanetJson.put("succeedingTitles", new JsonArray().add(succeedingTitle));
+
+    IndividualResource createdInstance = instancesClient.create(smallAngryPlanetJson);
+
+    JsonObject newSucceedingTitle = new JsonObject()
+      .put("title", "New title")
+      .put("hrid", "inst000000000022")
+      .put("identifiers", new JsonArray().add(
+        new JsonObject()
+          .put("identifierTypeId", "8261054f-be78-422d-bd51-4ed9f33c3422")
+          .put("value", "0262012103")));
+
+    JsonObject newInstance = createdInstance.getJson().copy();
+    newInstance.put("succeedingTitles", new JsonArray().add(newSucceedingTitle));
+
+    instancesClient.replace(createdInstance.getId(), newInstance);
+
+    Response instanceResponse = instancesClient.getById(createdInstance.getId());
+    assertPrecedingSucceedingTitles(instanceResponse.getJson().getJsonArray("succeedingTitles")
+        .getJsonObject(0), newSucceedingTitle, createdInstance.getId().toString(),
+      null);
+  }
+
+  @Test
+  public void canDeleteAnInstanceWithUnconnectedPrecedingTitles()
```

Comment:
Does this test demonstrate deleting the instance or removing the unconnected preceding title?

Reply:
Renamed